### ISSUE TITLE
Text Editor Cursor Position

### ISSUE THREAD

**Issue by aalpgiray** [nhn/tui.image-editor#325]
## Version
3.7.3

## Development Environment
Windows 10, Chrome 

## Current Behavior
The text editor cursor position is wrong when you are compiling a long sentence and click in the middle of a word. This behaviour can be observed on the [demo page](https://ui.toast.com/tui-image-editor/)

The screenshot below is demonstrating the issue. The cursor is placed almost in between two characters. But, in reality, it should be placed after the letter t. This gets worse on some occasion and it is frustrating if you are trying to compose somewhat longer sentences.

![image](https://user-images.githubusercontent.com/6337113/73918644-0ce73600-48d3-11ea-90dc-29d789f1cda0.png)

## Expected Behavior
Should work like any other text edit tool. The cursor should be on the right side of the last character.

**Comment by jinwoo-kim-nhn** [comment]
hi @aalpgiray 

Thanks for the bug report.

**Comment by ahsannajam** [comment]
> hi @aalpgiray
> 
> Thanks for the bug report.

Is this issue resolved?

**Comment by aalpgiray** [comment]
Yeah, I have checked the demo page again, and it seems the problem resolved!!! :D Now I can find the related backlog item in our issue tracker :D

**Comment by ahsannajam** [comment]
Brother issue not resolved yet. I have updated the editor to the version 3.10.0 but no impact. Whenever we add a long text it disturbs the blinking cursor position
